MANIFEST.in

```diff
@@ -2,6 +2,7 @@ include *.c
 include *.h
 include *.in
 include *.md
+include *.patch
```

Comment:
If you `graft patches`, then this isn't needed - see https://github.com/freakboy3742/Pillow/pull/3